docs/dev/fundamentals/rollups.md

```diff
@@ -0,0 +1,37 @@
+# Introduction
+
+To better understand rollups we need to dive briefly into Ethereum and Layer 2 solutions.
+
+The enormous demands from users to transact Ethereum put a limit on the network capacity by causing congestion, resulting in slow transactions and increased gas prices. While this has remained so for a while, an improved solution is needed, one that can maintain scalability.
+That is, to increase finality and achieve high transaction rates per second without having to trade off security. That's where layer 2 solutions shine.
```

Comment:
> to increase finality

what does it mean?

rates -> rate?

Reply:
yeah

Reply:
> transaction rates

And still there is "rate"

Reply:
What do you imply @StanislavBreadless

Reply:
> high transaction rates per

@barakshani in the comment above proposed to change "rates" to "rate" and you agreed, but the "transaction rates" is still there

Reply:
Please accept this suggestion:

```suggestion
That is, to increase finality and achieve high transaction rate per second without having to trade off security. That's where layer 2 solutions shine.